docs/core/porting/libraries.md

```diff
@@ -1,105 +1,113 @@
-title: Port libraries to .NET Core
-description: Learn how to port library projects from the .NET Framework to .NET Core.
+title: Port libraries to .NET
+description: Learn how to port library projects from the .NET Framework to .NET.
 author: cartermp
-ms.date: 12/07/2018
+ms.date: 03/04/2021
 ---
-# Port .NET Framework libraries to .NET Core
+# Port .NET Framework libraries to .NET
 
-Learn how to port .NET Framework library code to .NET Core, where it runs cross-platform and expands the reach of the apps that use it.
+Learn how to port .NET Framework library code to .NET, where it runs cross-platform and expands the reach of the apps that use it.
 
 ## Prerequisites
 
 This article assumes that you:
 
-- Are using Visual Studio 2017 or later. (.NET Core isn't supported on earlier versions of Visual Studio.)
+- Are using Visual Studio 2019 or later?
+  - .NET 5 requires Visual Studio 2019 (v16.9) or later.
+  - .NET Core 3.1 requires Visual Studio 2019 (v16.4) or later.
 - Understand the [recommended porting process](index.md).
 - Have resolved any issues with [third-party dependencies](third-party-deps.md).
 
-You should also become familiar with the content of the following articles:
+Familiarize yourself with the content of the following articles:
 
-[.NET Standard](../../standard/net-standard.md)\
+- [.NET Standard.](../../standard/net-standard.md)\
 This article describes the formal specification of .NET APIs that are intended to be available on all .NET implementations.
 
-[Developing Libraries with Cross Platform Tools](../tutorials/libraries.md)\
-This article explains how to write libraries using the .NET Core CLI.
+- [Develop libraries with the .NET CLI.](../tutorials/libraries.md)\
+This article explains how to write libraries using the .NET CLI.
 
-[.NET project SDKs](../project-sdk/overview.md)\
+- [.NET project SDKs.](../project-sdk/overview.md)\
 This article describes the SDK-style project file format.
 
-[Porting to .NET Core - Analyzing your Third-Party Party Dependencies](third-party-deps.md)\
-This article discusses the portability of third-party dependencies and what to do when a NuGet package dependency doesn't run on .NET Core.
+- [Analyze your dependencies to port code from .NET Framework to .NET.](third-party-deps.md)\
+This article discusses the portability of third-party dependencies, and what to do when a NuGet package dependency doesn't run on .NET.
 
 ## Retarget to .NET Framework 4.7.2
 
 If your code isn't targeting .NET Framework 4.7.2, we recommended that you retarget to .NET Framework 4.7.2. This ensures the availability of the latest API alternatives for cases where .NET Standard doesn't support existing APIs.
 
 For each of the projects you wish to port, do the following in Visual Studio:
 
-1. Right-click on the project and select **Properties**.
-1. In the **Target Framework** dropdown, select **.NET Framework 4.7.2**.
-1. Recompile the project.
+01. Right-click on the project and select **Properties**.
```

Comment:
Curious on this: What's the difference between *01* and *1*?

Reply:
@BillWagner it aligns the first non-whitespace after the `.`, to the 4th character, easy for tabbing child content so they align correctly: ([commonmark spec dictates this rule](https://spec.commonmark.org/0.29/#list-items))

```markdown
1. This is 3rd column
    This is 4th column and technically wrong though the parser may ignore it for now
```
You could just use 3rd level indentation on child items, but I find this annoying when pasting things like code as a list item child

```markdown
1. This is 3rd column
   This is 3rd column and correct
```

You could use two spaces on the first to align correctly if you wanted

```markdown
1.  This is 4th column
    This is 4th column and correct, though I find two spaces in the parent line item easy to accidentally destroy
```

I find it easier to just use 01

```markdown
01. This is 4th column
    This is 4th column and correct
```